doc/quickstart.md

```
@@ -2,100 +2,102 @@
+3. Multus-cni is deployed as default CNI plugin, and there is a default CNI plugin (flannel, ovn-kubernetes etc.) available for Multus-cni.
 4. On RedHat Enterprise Linux and Ubuntu operating systems, the `rdma-core` package must be installed to support RDMA resource provisioning. On RedHat CoreOS the package installation is not required.
 
 ## Installation
 
-Make sure to have installed the Operator-SDK, as shown in its [install documentation](https://sdk.operatorframework.io/docs/installation/), and that the binaries are available in your \$PATH.
+### OpenShift
 
-Clone this GitHub repository.
+For OpenShift clusters, you can use the Operator Lifecycle Manager (OLM) or deploy manually:
 
+**Option 1: Using OLM (Recommended)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)